src/amo/components/SearchResult.js

```diff
@@ -22,6 +22,9 @@ class SearchResult extends React.Component {
           <section className="SearchResult-main">
             <img className="SearchResult-icon" src={result.icon_url} alt="" />
             <h2 className="SearchResult-heading">{result.name}</h2>
+            <h3 className="SearchResult-rating">
+              <span className="visually-hidden">4/5</span>
```

Comment:
Oh wait, the answer to your question is **yes**.

I'll do that and stub in a review number.